### PR TITLE
Move remainder of Angular app under frontend

### PR DESCRIPTION
**Pull request by myabc**
Templates, project infrastructure, configuration.

See 6ec5ca58, #2261.

Also handles:
https://community.openproject.org/work_packages/17538

Continuation of code reorganisation started in #2193. This PR will another step towards allowing us  to move the frontend app into a separate repo.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://feature-frontend-reorganisation-complete.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by 0xF013** [comment]
@myabc I think it happens in me 1.3 branch as well

**Comment by myabc** [comment]
@0xF013 I also added template caching in this branch (albeit with Webpack rather than gulp: see commit 6f17d51f). I guess the Protractor failure might be related to this (i.e. many fewer HTTP requests being made).

**Comment by myabc** [comment]
Also deals with:
https://community.openproject.org/work_packages/17538

File: app/assets/javascripts/application.js.erb [hunk 1]
```diff
@@ -10,21 +10,10 @@
...
-//= require moment-timezone/moment-timezone.js
-//= require moment-timezone/moment-timezone-data.js
-
-//= require jquery.atwho/dist/js/jquery.atwho.js
-//= require Caret.js/src/jquery.caret.js
-//= require jquery_noconflict
+
+//= require ./bundles/openproject-global
+
+//# require jquery_noconflict
```
Comment (0xF013) [inline app/assets/javascripts/application.js.erb:16]: is this commented out?

Comment (myabc) [inline app/assets/javascripts/application.js.erb:16]: @0xF013 it is indeed. In fact, we can probably safely remove `jquery_noconflict`. Webpack only exposes `jQuery` as a global.

**Pull request merged**